foyer/forcefield.py

```diff
@@ -606,7 +602,7 @@ def createSystem(self, topology, nonbondedMethod=NoCutoff,
 
         data = self._SystemData
         data.atoms = list(topology.atoms())
-        for atom in data.atoms:
+        for _ in data.atoms:
```

Comment:
Just curious, does this work because data.atoms is a generator?

Reply:
Are you asking about the `_` or the ability to iterate over this object?

You can iterator over it because it's an _iterable_. 

`_` is just a special python character to hold the last expression value

Reply:
LGTM suggested this change because the `atom` value is not used inside the loop, it could have just as well been something like `for NULL in range(len(data.atoms)):`

(although I don't understand the OpenMM API enough to follow what's happening here, I guess it's just appending num_atoms off empty lists)